packages/utils/package.json

```
@@ -33,11 +33,12 @@
         "react": "^16.14.0 || ^17.0.0",
         "react-dom": "^16.14.0 || ^17.0.0",
         "react-redux": ">=7.0.0",
-        "react-router-dom": "^5.0.0"
+        "react-router-dom": "^5.0.0",
+        "cypress": "^10.9.0"
```

```suggestion
        "cypress": ">=10.0.0 < 11.0.0"
```
please, so that we tolerate all 10.x versions

---

Done